source/blog/2019-01-06-ovirt-openshift-part-1.html.md

```diff
@@ -0,0 +1,148 @@
+
+
+
+<script id="asciicast-219956" src="https://asciinema.org/a/219956.js" async></script>
+
+## Single shell file installation
+
+Dropping to shell - this [install.sh](https://github.com/oVirt/ovirt-openshift-extensions/blob/master/automation/ci/install.sh) is a wrapper for installing  the ovirt-openshift-installer container, it uses ansible-playbook and has two main playbooks: install_okd.yaml and install_extensions.yaml. The latter is mainly for installing ovirt storage plugins.
+
+The install.sh script has one dependency, it needs to have 'podman' installed on the host, while all the rest runs inside a container.
+
+The only dependency (except from running oVirt datacenter) is podman:
+```console
+[bastion ~]# dnf install podman
+``` 
+
+[For other ways to install podman consult the readme](https://github.com/containers/libpod/blob/master/docs/tutorials/podman_tutorial.md)
+
+If you can't install `podman` docker will be fine as well, just edit the install.sh, and substitute podman for docker.
+
+### Get the install.sh and customize
+```console
+[bastion ~]# curl -O "https://raw.githubusercontent.com/oVirt/ovirt-openshift-extensions/master/automation/ci/{install.sh,customization.yaml}"
+```
+
+Edit the `customization.yaml`:
+
+- Put the engine details in engine_url
+  ```yaml
+  engine_url: https://ovirt-engine-fqdn/ovirt-engine/api
+  ```
+
+- Choose the oVirt cluster and data domain you want, if you don't want 'Default'
+  ```yaml
+  openshift_ovirt_cluster: yours
+  openshift_ovirt_data_store: yours
+   ```
+- Unmark the memory check if you don't change the VM default memory for this install is 8Gb
+  ```yaml
+  openshift_disable_check: memory_availability,disk_availability,docker_image_availability
+  ```
+
+- Domain name of the setup. The setup will create a VM with the name master0.$public_hosted_zone here. This VM will
+  be used for all the components of the setup
+  ```yaml
+  public_hosted_zone: example.com
+  ```
+
+For a more complete list of customizations, (take a look at the vars.yaml](https://github.com/oVirt/ovirt-openshift-extensions/blob/master/automation/ci/vars.yaml) packaged in the container.
```

Comment:
Something is wrong here, maybe ( instead of [

Reply:
done